Makefile

```diff
@@ -71,6 +71,13 @@ proto: ## Rebuild protobuf files
 	go run scripts/protobuf.go install
 	go run scripts/protobuf.go generate
 
+.PHONY: build-packages
+build-packages: build-packages-race build-packages-normal ## Test docker images locally
```

Comment:
Aren't both targets saving the binaries in the same directory, so one overrides the other?

Reply:
Actually, I don't care about the binaries, it's only to build the packages and cache them, such that compilation and linters run faster later on.